### PR TITLE
Feature/input/output in edit text

### PR DESCRIPTION
**Pull request by TaniaPichka**
All difits and characters (+, -, *, :, .) are clickable except "C" button.

### CODE REVIEW

File: Calculator/app/src/main/java/com/example/user/calculator/MainActivity.java [hunk 1]
```diff
@@ -11,42 +11,126 @@
 
 public class MainActivity extends ActionBarActivity {
 
-   // EditText editText;
-   // Button num7;
-   // Button num8;
-   // Button num9;
+    EditText editText;
```
Comment (DeadDogDeus) [inline Calculator/app/src/main/java/com/example/user/calculator/MainActivity.java]: Use property and add access modifier for each

File: Calculator/app/src/main/java/com/example/user/calculator/MainActivity.java [hunk 2]
```diff
@@ -11,42 +11,126 @@
...
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.main);
 
-       // editText = (EditText) findViewById(R.id.editText);
-       // num7 = (Button) findViewById(R.id.num7);
-       // num8 = (Button) findViewById(R.id.num8);
-       // num9 = (Button) findViewById(R.id.num9);
+        editText = (EditText) findViewById(R.id.editText);
```
Comment (DeadDogDeus) [inline Calculator/app/src/main/java/com/example/user/calculator/MainActivity.java]: Move it to separate function

File: Calculator/app/src/main/java/com/example/user/calculator/MainActivity.java [hunk 3]
```diff
@@ -11,42 +11,126 @@
...
+        div = (Button) findViewById(R.id.div);
+        sub = (Button) findViewById(R.id.sub);
+        mul = (Button) findViewById(R.id.mul);
+        dot = (Button) findViewById(R.id.dot);
+        equallyButton = (Button) findViewById(R.id.equallyButton);
 
-       /* View.OnClickListener onClickListener = new View.OnClickListener() {
+        //  input/output in EditText
+
+        View.OnClickListener onClickListener = new View.OnClickListener() {
```
Comment (DeadDogDeus) [inline Calculator/app/src/main/java/com/example/user/calculator/MainActivity.java]: Could we decrease size of this method?

**Comment by DeadDogDeus** [comment]
Use gitignore file

**Comment by TaniaPichka** [comment]
Method for each button was added.
Modifiers for all variables was added.

**Pull request merged**